modules/migration-creating-migration-plan-cam.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// migration/migrating_openshift_3_to_4/migrating-openshift-3-to-4.adoc
+[id='migration-creating-migration-plan-cam_{context}']
+= Creating a migration plan in the CAM web console
+
+.Prerequisites
+
+* Source and target clusters added to the CAM web console
```

Comment:
Need to either say "have been added" or reword to something like "You have added the source and target clusters to the CAM web console."

Reply:
Done. Did the same for the storage.